clients/ordering/public/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script src="https://js.stripe.com/v3/"></script>
```

Comment:
Just to verify, is this necessary? Can we add it as a node dependency instead?

Reply:
Not sure, but it wasn't functioning without it.

Reply:
Ok. I read the docs for react-stripe-elements and this is fine.